opendut-carl/carl-development.toml

```diff
@@ -13,3 +13,8 @@ ui.directory = "./opendut-lea/dist/"
 
 [vpn]
 enabled = false
+
+[opentelemetry]
+enabled = true
+endpoint = "http://localhost:4317"
+service_name = "OTel"
```

Comment:
You can remove the `service_name` config here, because the default set in `carl.toml` is sufficient.